com.hudren.woodpile/src/com/hudren/woodpile/model/LogEvent.java

```diff
@@ -47,6 +55,14 @@
 
 	private static final ConcurrentHashMap<String, String> HOSTS = new ConcurrentHashMap<String, String>();
 
+	/**
+	 * Helper converters for Logback events.
+	 */
+	private static final ClassOfCallerConverter classNameConverter = new ClassOfCallerConverter();
```

Comment:
I wasn't sure of the best place to put these. This seemed like a reasonable place until a better one presents itself.

Reply:
This is fine.